src/includes/set-context/dart.mdx

```diff
@@ -1,9 +1,10 @@
 ```dart
 import 'package:sentry/sentry.dart';
 
-final person = {
+final character = {
   'name': 'Mighty Fighter',
   'age': '19',
```

Comment:
```suggestion
  'age': 19,
```

@marandaneto would this still be correct if it was a number? Other examples use an integer instead of string here.

Reply:
Assuming yes here.